scripts/playout_controls.sh

```diff
@@ -112,6 +112,21 @@ if [ "${DEBUG_playout_controls_sh}" == "TRUE" ]; then echo "VAR VALUE: $VALUE" >
 
 case $COMMAND in 
     shutdown)
+        while :
+        do
+            apt=1
+            sudo lsof /var/lib/apt/lists/lock > /dev/null
```

Comment:
all 3 are necessary. not perfectly sure about the last one as it might be a dupe of the first but 1+2 still different though. you can install via dpkg without using apt. better check 1+2 and to be sure 3. i might be extra cautious after my issue as it wasted a lot of time i would have loved to spend differently.